scan/lib/scan/test_command_generator.rb

```diff
@@ -85,9 +85,9 @@ def pipe
 
       formatter = []
       if Scan.config[:formatter]
-        formatter << "-f `#{Scan.config[:formatter]}`"
+        formatter << "-f '#{Scan.config[:formatter]}'"
       elsif FastlaneCore::Env.truthy?("TRAVIS")
-        formatter << "-f `xcpretty-travis-formatter`"
+        formatter << "-f 'xcpretty-travis-formatter'"
```

Comment:
The backticks are actually needed if you are loading a custom formatter as a gem (which is what this is doing so this needs to stay as backticks)